<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="A-MACBOOKPRO-1.local" name="com.balfour.sb4.unittests.RequestPwordEmail_Test" tests="1" failures="1" timestamp="9 Feb 2015 16:45:51 GMT" time="22.161" errors="0">
  <testcase name="requestPwordEmail_Test" time="22.161" classname="com.balfour.sb4.unittests.RequestPwordEmail_Test">
    <failure type="java.lang.RuntimeException" message="file not present">
      <![CDATA[java.lang.RuntimeException: file not present
	at com.balfour.publishing.qa.pages.sb4.SshUtil.lsFile(SshUtil.java:84)
	at com.balfour.publishing.qa.pages.sb4.Sb4LoginPage.chkEmailLogFile(Sb4LoginPage.java:159)
	at com.balfour.sb4.unittests.RequestPwordEmail_Test.requestPwordEmail_Test(RequestPwordEmail_Test.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- requestPwordEmail_Test -->
</testsuite> <!-- com.balfour.sb4.unittests.RequestPwordEmail_Test -->
